addons/project_timesheet_holidays/models/resource_calendar_leaves.py

```diff
@@ -111,13 +127,13 @@ def _timesheet_create_lines(self):
             ] for line in holidays_read_group
         }
         vals_list = []
-        for leave in self:
-            for employee in mapped_employee.get(leave.calendar_id.id, self.env['hr.employee']):
+        def get_timesheets_data(employees, work_hours_list):
```

Comment:
Can you add a blank line above and bottom the function to let the function breathing a little bit? 😄 

Thanks in advance! 🙂